cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -133,11 +216,21 @@ int32_t cros_gralloc_driver::allocate(const struct cros_gralloc_buffer_descripto
 	uint64_t use_flags;
 	int32_t reserved_region_fd;
 	char *name;
+	bool from_kms = false;
 
 	struct bo *bo;
 	struct cros_gralloc_handle *hnd;
 
-	resolved_format = drv_resolve_format(drv_, descriptor->drm_format, descriptor->use_flags);
+	struct driver *drv;
+
+	if ((descriptor->use_flags & BO_USE_SCANOUT)) {
+		from_kms = true;
+		drv = drv_kms_;
+	} else {
+		drv = drv_render_;
+	}
+
```

Comment:
Can we add format & usage check for kmsro case? Then if there is one issue, we can get right error return in time.